docs/tutorial_digital_earth_pacific_patch_level.ipynb

```diff
@@ -0,0 +1,1311 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "287c93e9",
+   "metadata": {},
+   "source": [
+    "# Digital Earth Pacific mineral resource detection using Clay\n",
+    "\n",
+    "This notebook applies the Clay model on imagery composites, specifically a Sentinel-2\n",
+    "Geometric Median [(GeoMAD)](https://docs.digitalearthafrica.org/en/latest/data_specs/GeoMAD_specs.html) composite. We will use\n",
+    "[Digital Earth Pacific's STAC API](https://stac-browser.staging.digitalearthpacific.org)\n",
+    "to obtain these datasets, and apply it on a mineral resource detection downstream task to do the following:\n",
+    "\n",
+    "1. Generate fine level (pixel of size 10m x 10m) embeddings for an area (5.12km x 5.12km) where known mining extraction is occurring\n",
+    "2. Save the fine level (patch) embeddings and execute a similarity search that leverages the ground truth points as reference\n",
+    "\n",
+    "We will also leverage the [AWS Sentinel-2 STAC catalog](https://registry.opendata.aws/sentinel-2/) to obtain fine \"patch\" level cloud cover percentages from the [Scene classification (SCL) mask](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/). These percentages will be mapped and added to the GeoParquet files such that they can be added to database tables for similarity search filters and any other relevant downstream tasks.\n",
```

Comment:
Mixing the cloud-free composite from DEP'S STAC with the SCL mask from AWS's Sentinel 2 STAC seems... weird? I see that you're trying to consolidate the notebook from #184 with the one here, but there's an inherent mismatch with the data from two different STAC catalogs. It won't make sense to do a cloud-percentage pre-filter using the SCL band, when the composite image is technically cloud-free already.